.credo.exs

```diff
@@ -30,10 +30,11 @@
         {Credo.Check.Readability.TrailingBlankLine},
         {Credo.Check.Readability.TrailingWhiteSpace},
         {Credo.Check.Readability.VariableNames},
-        {Credo.Check.Refactor.ABCSize},
+        {Credo.Check.Refactor.ABCSize, max_size: 40},
         {Credo.Check.Refactor.CaseTrivialMatches},
         {Credo.Check.Refactor.CondStatements},
         {Credo.Check.Refactor.FunctionArity},
+        {Credo.Check.Refactor.MapInto, false},
```

Comment:
We shouldn’t add a disabled check 😄

Reply:
It raises a warning on Elixir 1.8

Reply:
It’s enabled by default, but throws a warning when running on `1.8`